src/modules/user_certificate/dto/create-user-course-certificate.dto.ts

```
@@ -0,0 +1,42 @@
+import { IsString, IsUUID, IsOptional, IsInt, IsDate } from 'class-validator';
```

ESLint

[error] 'IsDate' is defined but never used.

(@typescript-eslint/no-unused-vars)